src/tribol/interface/tribol.cpp

```diff
@@ -921,6 +1110,96 @@ integer update( integer cycle, real t, real &dt )
       }
 
       CouplingScheme* couplingScheme  = csManager.getCoupling(csIndex);
+
+      // update redecomp meshes if supplied mfem data
+      if (couplingScheme->hasMfemData())
+      {
+         auto mfem_data = couplingScheme->getMfemMeshData();
+         axom::Array<int> mesh_ids {2, 2};
```

Comment:
Is {2,2} correct?

Reply:
Yes, it's calling the constructor which sets the size and capacity to 2.